teku/src/main/java/tech/pegasys/teku/cli/BeaconNodeCommand.java

```diff
@@ -134,50 +131,44 @@ private static class ConfigFileCommand {
       scope = ScopeType.INHERIT)
   private File configFile;
 
-  @Mixin(name = "Network")
-  private Eth2NetworkOptions eth2NetworkOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Network%n")
+  private Eth2NetworkOptions eth2NetworkOptions = new Eth2NetworkOptions();
 
-  @Mixin(name = "P2P")
-  private P2POptions p2POptions;
+  @CommandLine.ArgGroup(validate = false, heading = "P2P%n")
+  private P2POptions p2POptions = new P2POptions();
 
-  @Mixin(name = "Interop")
-  private InteropOptions interopOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator%n")
+  private ValidatorOptions validatorOptions = new ValidatorOptions();
 
-  @Mixin(name = "Validator")
-  private ValidatorOptions validatorOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Execution Layer%n")
+  private ExecutionLayerOptions executionLayerOptions = new ExecutionLayerOptions();
 
-  @Mixin(name = "Deposit")
-  private DepositOptions depositOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Beacon Node Data%n")
+  private BeaconNodeDataOptions dataOptions = new BeaconNodeDataOptions();
 
-  @Mixin(name = "Execution Layer")
-  private ExecutionLayerOptions executionLayerOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Data Storage%n")
+  private DataStorageOptions dataStorageOptions = new DataStorageOptions();
 
-  @Mixin(name = "Logging")
-  private final LoggingOptions loggingOptions = new LoggingOptions();
-
-  @Mixin(name = "Metrics")
-  private MetricsOptions metricsOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Beacon REST API%n")
+  private BeaconRestApiOptions beaconRestApiOptions = new BeaconRestApiOptions();
 
-  @Mixin(name = "Data")
-  private BeaconNodeDataOptions dataOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Validator REST API%n")
+  private ValidatorRestApiOptions validatorRestApiOptions = new ValidatorRestApiOptions();
 
-  @Mixin(name = "Data Storage")
-  private DataStorageOptions dataStorageOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Weak Subjectivity%n")
+  private WeakSubjectivityOptions weakSubjectivityOptions = new WeakSubjectivityOptions();
 
-  @Mixin(name = "Store")
-  private StoreOptions storeOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Interop Options%n")
+  private InteropOptions interopOptions = new InteropOptions();
 
-  @Mixin(name = "Beacon REST API")
-  private BeaconRestApiOptions beaconRestApiOptions;
+  @CommandLine.ArgGroup(validate = false, heading = "Store Options%n")
+  private final StoreOptions storeOptions = new StoreOptions();
```

Comment:
Interop Options and Store Options don't have any options available in `--help`, since they are all unstable and set to hidden. The headings will still show up and there doesn't appear to be an option to conditionally hide them if empty. Should we leave that as is, or suggestions on what I can do here?

Reply:
It's a bit messy and maybe we should find better places to move those options to (though `InteropOptions` is pretty spot on) but we can use a `Mixin` for these and just add support for picking up arg groups and mixins in `UnstableOptionsCommand`. I'll push that change up shortly since I had to fiddle around to see if it was possible anyway.